src/lines.rs

```diff
@@ -32,7 +32,7 @@ pin_project! {
 impl<R> Lines<R> {
     pub(crate) fn new(reader: R) -> Lines<R>
     where
-        R: AsyncBufRead + Unpin + Sized,
+        R: AsyncBufRead + Sized,
```

Comment:
I removed `Unpin` but I'm not sure if this is correct.
Does `reader: R` actually need to implement `Unpin`...?